entrypoint.sh

```diff
@@ -14,10 +18,12 @@ fi
 # --- Determine docker host address ------------------------------------------
 
 function _resolve_host {
-  getent ahostsv4 "$1" | head -n1 | cut -d' ' -f1
+  ip=$(getent ahostsv4 "$1" | head -n1 | cut -d' ' -f1)
+  echo "$ip"
 }
 
-if [ "$DOCKER_HOST" ]
+# Check if the docker host env var is set
+if [ ! -z "${DOCKER_HOST+x}" ]
```

Comment:
what does the `+x` do?

Reply:
This tests if the `DOCKER_HOST` env var is set, and if not, substitutes the string x otherwise. This allows me to distinguish between an env var that is set, and one that is set to an empty string. But I realize now that I don't need the `+x`, because an empty string value for `$DOCKER_HOST` is just as useless (I copied this from one of my personal scripts). So I'll remove the `+x` part.

Reply:
btw `[ ! -z "$DOCKER_HOST" ]` is equals to `[ -n "$DOCKER_HOST" ]` is equals to `[ "$DOCKER_HOST" ]` so you can revert this  change as well

Reply:
You are correct indeed! I don't know why I didn't think of this, it's even done in the script as well. I'll change it back so it's more consistent and nicer.